Makefile.tools

```diff
@@ -20,9 +24,18 @@ verilator_build:
 	$(MAKE) -C $(verilator_dir)
 	$(MAKE) -C $(verilator_dir) install
 
+dromajo_build:
+	mkdir -p $(dromajo_dir)/build
+	$(CMAKE) -S $(dromajo_dir) -B $(dromajo_dir)/build
```

Comment:
`$(CMAKE) -S $(dromajo_dir) -B $(dromajo_dir)/build -DCMAKE_BUILD_TYPE=Release`
Small change to make dromajo with -O3.

Reply:
cool!